stdlib/2/UserString.pyi

```diff
@@ -5,6 +5,7 @@ _UST = TypeVar("_UST", bound=UserString)
 _MST = TypeVar("_MST", bound=MutableString)
 
 class UserString(Sequence[UserString]):
+    data: Union[str, unicode]
```

Comment:
Since `unicode` is a super type of `str` (for type checking purposes), this should just be `data: unicode`.

Reply:
Ah right, I didn't know that. I've added a new commit with the fix, thanks!